black/Dockerfile

```diff
@@ -1,10 +1,18 @@
+FROM python:3-alpine as black-build
```

Comment:
A multi-stage build might be overkill here, you can do something like:

```
RUN set -ex; \
         apk add --no-cache build-base; \
         pip install --upgrade black; \
         apk del build-base

Reply:
Thanks for the comment. I've modified my PR and converted it back to a single stage for clarity.